### DISCUSSION TITLE
VRAM Increase in HD4000, Any possibilities and benefits?

### DISCUSSION THREAD

**Discussion by Mtkhan13** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#205] [Q&A]
Hello Chris, Thanks for the great work.
I have a question as I've a EB 8470p with i7-3740QM upgraded. I wanted to ask, is there a way I can increase VRAM of HD4000? currently showing 1536 MB. If yes would it be helpful improving the graphics performance or it will be just a cosmetic change? please suggest if any solution exist? I will be also upgrading the RAM to 16GB soon in couple of days. 
![Screenshot 2024-04-11 at 5 31 36 PM](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/22547975/c877e314-33f5-4cd3-b04c-facbfc8959fc)
![Screenshot 2024-04-11 at 5 49 58 PM](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/22547975/afc48ed9-c94f-4496-87bf-3d3714368af7)

**Comment by Mtkhan13** [comment]
current OS 10.15 with your OC boot loader nothing modified.

**Comment by chris1111** [comment]
4 gig of ram=1024 mb
8 gig of ram=1536 mb
16 gig of ram=2048 mb
Thanks

**Comment by Mtkhan13** [comment]
Hi friend sorry for late reponse. 
I just upgraded the RAM but it didn't increase the GPU VRAM 
![image](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/22547975/d5b1fbb7-56f6-4d0b-8172-d4409bb83d2b)
How can I do that?

**Comment by Mtkhan13** [comment]
Hi my friend I'm still waiting for your reply.

**Comment by chris1111** [comment]
Strange?
Have a look [here](https://dortania.github.io/OpenCore-Post-Install/gpu-patching/intel-patching/vram.html#creating-our-patch) Then [here](https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md)

**Comment by Mtkhan13** [comment]
Do you think it only changes while installing the mac? if its the case then I will have to reinstall the mac.

**Reply by chris1111** [reply]
I dont know?
